### PR TITLE
EZP-32111: Added ContentTranslatedName sort clause

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: [EZP-32111](https://jira.ez.no/browse/EZP-32111)

### Description 

Impl. visitor for https://github.com/ezsystems/ezplatform-kernel/pull/143

## Checklist

- [X] Code follows the code style of this project (use `$ composer fix-cs`).
- [x] Change requires a change to the documentation.
- [x] Code is ready for a review.
- [x] Drop 3bc6660f6859d9b53666a092112f912fea3926d6 before merge

### CODE REVIEW

File: lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public function canVisit(SortClause $sortClause): bool
+    {
+        return $sortClause instanceof SortClause\ContentTranslatedName;
+    }
+
+    public function visit(SortClause $sortClause): string
+    {
+        return 'meta_content__name_s' . $this->getDirection($sortClause);
+    }
+}
```
Comment (Steveb-p) [inline lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php:25]: Missing newline.

**Review by Steveb-p — APPROVED** [review]
Add simple test for `ContentTranslatedName` if possible.

File: lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php (hunk 1 above)
Comment (adamwojs) [reply lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php]: Fixed in ebd7985

**Review by ViniTou — APPROVED** [review]

File: lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+final class ContentTranslatedName extends SortClauseVisitor
+{
+    public function canVisit(SortClause $sortClause): bool
+    {
+        return $sortClause instanceof SortClause\ContentTranslatedName;
+    }
+
+    public function visit(SortClause $sortClause): string
+    {
+        return 'meta_content__name_s' . $this->getDirection($sortClause);
```
Comment (mikadamczyk) [inline lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php:23]:
```suggestion
        return 'meta_content_name_s' . $this->getDirection($sortClause);
```

**Review by mikadamczyk — APPROVED** [review]

File: lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php (hunk 2 above)
Comment (adamwojs) [reply lib/Query/Common/SortClauseVisitor/ContentTranslatedName.php:23]: It's not a mistake. See https://github.com/ezsystems/ezplatform-solr-search-engine/blob/master/lib/FieldMapper/ContentTranslationFieldMapper/ContentDocumentTranslatedContentNameField.php#L28

**Review by alongosz — APPROVED** [review]

**Pull request merged**